zebedee-reader/src/main/java/com/github/onsdigital/zebedee/content/base/ContentLanguage.java

```diff
@@ -0,0 +1,42 @@
+package com.github.onsdigital.zebedee.content.base;
+
+import java.util.stream.Stream;
+
+public enum ContentLanguage {
+    ENGLISH("en"), WELSH("cy");
```

Comment:
Just wondering if these constants should be `EN` and `CY`. Not objecting to them being what they are, just raising the thought. I'm happy to go with whatever you decide.

Reply:
I thought I'd rename it to use the full word for easier reading but it is only naming and can be changed back to `EN` and `CY` if preferred